.gitignore

```diff
@@ -3,6 +3,7 @@
 /.project
 /.vagrant
 .vscode/
+.idea/
```

Comment:
This should be in your global gitignore on your workstation, not here. Go ahead and remove the .vscode one also.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Thanks @coreydaley for providing the information